packit_service/worker/helpers/build/build_helper.py

```diff
@@ -314,14 +326,22 @@ def tests_targets_for_test_job(self, test_job_config: JobConfig) -> set[str]:
 
         if self.build_targets_override:
             logger.debug(f"Build targets override: {self.build_targets_override}")
-            for target in self.build_targets_override:
-                targets_override.update(
-                    self.build_target2test_targets_for_test_job(target, test_job_config),
-                )
+            for target, identifier in self.build_targets_override:
+                if identifier == (test_job_config.identifier if test_job_config.identifier else ""):
```

Comment:
Why do you need the ternary there? :eyes:

Reply:
`job_config.identifier` could be `None`, so yes I think we need it. A non specified `identifier` is the default identifier "".
We can also say that the default identifier is always `None` but in this case I am a bit afraid of the serialization, if we go through a serialization/deserialization process of a `None` string value I think we will end up with an empty str ""? 
So I feel this comparison more safe.